gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/handler/HttpGremlinEndpointHandler.java

```diff
@@ -262,7 +269,7 @@ public void channelRead(final ChannelHandlerContext ctx, final Object msg) {
                             attemptCommit(requestMessage.getArg(Tokens.ARGS_ALIASES), graphManager, settings.strictTransactionManagement);
 
                             try {
-                                return Unpooled.wrappedBuffer(serializer.getValue1().serializeResponseAsString(responseMessage, ctx.alloc()).getBytes(UTF8));
+                                return Unpooled.wrappedBuffer(serializer.getValue1().serializeResponseAsBinary(responseMessage, ctx.alloc()));
```

Comment:
i don't remember why it was chosen to encode this way (and perhaps it was wrong to do) but this seems like a breaking change.  The following is from 3.7.0 and would no longer return the same response, right?:

```
$ curl -v -H Accept:application/vnd.graphbinary-v1.0 http://localhost:8182?gremlin=g.V%28%29
*   Trying 127.0.0.1:8182...
* Connected to localhost (127.0.0.1) port 8182 (#0)
> GET /?gremlin=g.V%28%29 HTTP/1.1
> Host: localhost:8182
> User-Agent: curl/7.81.0
> Accept:application/vnd.graphbinary-v1.0
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< content-type: application/vnd.graphbinary-v1.0
< content-length: 872
< 
* Connection #0 to host localhost left intact
gQDX0dKWp4pPrJQSEq6fq21cAAAAyAAAAAAAAAAAAAAAAAAJAAAAAAYRAAIAAAAAAAAAAAEAAAAGcGVyc29uCQAAAAACEgACAAAAAAAAAAAAAAAABG5hbWUDAAAAAAVtYXJrb/4BCQAAAAAAEgACAAAAAAAAAAABAAAAA2FnZQEAAAAAHf4BCQAAAAAAEQACAAAAAAAAAAACAAAABnBlcnNvbgkAAAAAAhIAAgAAAAAAAAAAAgAAAARuYW1lAwAAAAAFdmFkYXP+AQkAAAAAABIAAgAAAAAAAAAAAwAAAANhZ2UBAAAAABv+AQkAAAAAABEAAgAAAAAAAAAAAwAAAAhzb2Z0d2FyZQkAAAAAAhIAAgAAAAAAAAAABAAAAARuYW1lAwAAAAADbG9w/gEJAAAAAAASAAIAAAAAAAAAAAUAAAAEbGFuZwMAAAAABGphdmH+AQkAAAAAABEAAgAAAAAAAAAABAAAAAZwZXJzb24JAAAAAAISAAIAAAAAAAAAAAYAAAAEbmFtZQMAAAAABGpvc2j+AQkAAAAAABIAAgAAAAAAAAAABwAAAANhZ2UBAAAAACD+AQkAAAAAABEAAgAAAAAAAAAABQAAAAhzb2Z0d2FyZQkAAAAAAhIAAgAAAAAAAAAACAAAAARuYW1lAwAAAAAGcmlwcGxl/gEJAAAAAAASAAIAAAAAAAAAAAkAAAAEbGFuZwMAAAAABGphdmH+AQkAAAAAABEAAgAAAAAAAAAABgAAAAZwZXJzb24JAAAAAAISAAIAAAAAAAAAAAoAAAAEbmFtZQMAAAAABXBldGVy/gEJAAAAAAASAAIAAAAAAAAAAAsAAAADYWdlAQAAAAAj/gEJAAAAAAA=
```

Reply:
$ curl -v -H Accept:application/vnd.graphbinary-v1.0 http://localhost:45940?gremlin=gmodern.V%28%29
* processing: http://localhost:45940?gremlin=gmodern.V%28%29
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying [::1]:45940...
* Connected to localhost (::1) port 45940
> GET /?gremlin=gmodern.V%28%29 HTTP/1.1
> Host: localhost:45940
> User-Agent: curl/8.2.1
> Accept:application/vnd.graphbinary-v1.0
>
< HTTP/1.1 200 OK
< content-type: application/vnd.graphbinary-v1.0
< content-length: 629
<
{ [629 bytes data]